upgrade.yml

```diff
@@ -4,6 +4,13 @@
 #
 # Assumes ml2 networking
 ---
+- name: Upgrade kernel to supported HWE
+  apt: pkg={{ item }}
+  with_items:
+    - linux-generic-lts-trusty
+    - linux-tools-lts-trusty
+  when: ansible_distribution_version == "12.04"
```

Comment:
You've put a task in place of a play, this isn't going to work, and it breaks upgrades.